bin/deploy.sh

```diff
@@ -5,6 +5,9 @@ DEPLOY_CMD="ansible-playbook deploy.yml -e env=$1 -e site=$2"
 ENVIRONMENTS=( hosts/* )
 ENVIRONMENTS=( "${ENVIRONMENTS[@]##*/}" )
 NUM_ARGS=2
+BRANCH_NAME="$(git symbolic-ref HEAD 2>/dev/null)" ||
+BRANCH_NAME="(unnamed branch)"     # detached HEAD
+BRANCH_NAME=${BRANCH_NAME##refs/heads/}
```

Comment:
Think we can simplify this a bit more:

```bash
BRANCH_NAME="$(git rev-parse --abbrev-ref HEAD)"
```

Nothing else should be necessary since we're only checking if it's not `master`.